src/app/api/posts/route.ts

```
@@ -0,0 +1,26 @@
+import { NextResponse } from "next/server";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+export async function GET() {
+  try {
+    const posts = await prisma.post.findMany();
+    return NextResponse.json(posts);
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)